src/project-2.js

```diff
@@ -44,62 +58,88 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num < 2) return false;
+  for (let i = 2; i < num; i++) {
+    return !((num % i) === 0);
+  }
+  return true;
 };
 
 const returnFirst = (arr) => {
   // return the first item from the array
+  return arr[0];
 };
 
 const returnLast = (arr) => {
```

Comment:
Here, you're mutating your array.
You'll want to be using `arr[arr.length - 1]`.